assets/scripts/src/choices.js

```diff
@@ -1921,11 +1953,6 @@ class Choices {
 
     // If target is something that concerns us
     if (this.containerOuter.contains(target)) {
-      // Handle button delete
```

Comment:
Hmm, why is this being removed? 🤔

Reply:
This code block threw an error because the delete is also handled in the onMouseDown. Removing this code block did not seem to cause errors (i could still click the delete button).

Reply:
I have a funny feeling it's there for when somebody is on a touch device. It may just be unnecessary code though!